### PR TITLE
Fix/tao 7845/browser zoom behavior for touchscreen

### PR DESCRIPTION
**Pull request by start-admin**
https://oat-sa.atlassian.net/browse/TAO-7845

_This is a fix for the Calculator zoom behavior._

Steps to reproduce:
Invoke calculator > Zoom in > after typing in each number, control goes to the part where the result is displayed.

**How to test on touch devices:**

- open https://act-test.taocloud.org
- find delivery http://localhost/tao.rdf#i1503426381071581600217278 (just type this in the search box in the delivery) or anyone else with Calculator.
- open delivery via lti (http://ltiapps.net/test/tc.php)
- Invoke calculator > Zoom in > after typing in each number, control goes to the part where the result is displayed.

**cause of the problem:**

When you click on the numbers, the autofocus on the textarea is triggered and because of this, the screen takes the user to this textarea.

**solution to the problem:**

autofocus is disabled for mobile devices because it is not needed there.

### CODE REVIEW

**Comment by krampstudio** [comment]
The Pull Request description is missing the required informations : 
```
  -  a link to the task
 -   a short summary of the pull request
  -  a complete description of what has been changed and why
  - how to test the Pull Request and steps to reproduce if the Pull Request fix a bug
  -  if the Pull Request has some dependencies to other Pull Request, they must be listed
```

**Comment by start-admin** [comment]
i am sorry you just left comments to the task and I thought you had already seen my previous pull request ( https://github.com/oat-sa/extension-tao-act/pull/1157 )

**Comment by start-admin** [comment]
i am sorry.  you just left comments to the task and I thought you had already seen my previous pull request. ( https://github.com/oat-sa/tao-core/pull/2075 )

File: views/js/lib/calculator/index.js [hunk 1]
```diff
@@ -16,17 +16,7 @@ define(['jquery', 'lodash', 'tpl!lib/calculator/template', 'i18n', 'lib/gamp/gam
...
-        })(navigator.userAgent||navigator.vendor||window.opera);
-
-        return check;
-    };
-
-    var isMobile = isMobileCheck();
+    var isTouchScreen = "ontouchstart" in window || navigator.maxTouchPoints
```
Comment (krampstudio) [inline views/js/lib/calculator/index.js]: the semi-colon is missing at the end of the line (ESLint should have warned you)

**Pull request merged**